packages/fiori/src/themes/ProductSwitch.css

```diff
@@ -0,0 +1,30 @@
+:host {
+	font-family: var(--sapUiFontFamily);
```

Comment:
Usually,  we write css regarding the html native 'hidden' attribute, so the user can hide our components, using it.

```css 
!-- from Button.css --!
:host(:not([hidden])) {
	display: inline-block;
}
```